common/remote_object.go

```diff
@@ -221,12 +219,12 @@ func parseRemoteObject(obj *cdpruntime.RemoteObject) (any, error) {
 	return nil, UnserializableValueError{uv}
 }
 
-func valueFromRemoteObject(ctx context.Context, robj *cdpruntime.RemoteObject) (goja.Value, error) {
+func valueFromRemoteObject(_ context.Context, robj *cdpruntime.RemoteObject) (any, error) {
 	val, err := parseRemoteObject(robj)
 	if val == "undefined" {
-		return goja.Undefined(), err
+		return nil, err
```

Comment:
This is the only change we needed to make to the remote object, so the tests can pass.